lxd/api_1.0.go

```diff
@@ -378,11 +380,10 @@ func doApi10UpdateTriggers(d *Daemon, nodeChanged, clusterChanged map[string]str
 			fallthrough
 		case "maas.api.key":
 			maasChanged = true
+		case "candid.api.expiry":
```

Comment:
Shouldn't that also set candidChanged then?

Reply:
It does: `fallthrough`

Reply:
Oh right, missed that somehow.